src/SankeyPlots.jl

```diff
@@ -167,18 +167,18 @@ In addition to [Plots.jl attributes](http://docs.juliaplots.org/latest/attribute
         @series begin
             primary := :false
             seriestype := :scatter
-            markersize := 0
+            markeralfa := 0
             series_annotations := text.(names, lab_orientations, label_size)
             xlabs, ylabs
         end
-        
+
         # extend axes for labels
         if label_position in (:left, :right)
             x_extra = label_position === :left ? minimum(xlabs) - 0.4 : maximum(xlabs) + 0.5
             @series begin
                 primary := false
                 seriestype := :scatter
-                markersize := 0
+                markeralfa := 0
```

Comment:
```suggestion
                markeralpha := 0
```

Reply:
ahahah sorry, sure! I copied the wrong text, sorry; I do have two copies and I copied the wrong one; in my source code the "alpha" one is used